src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -134,6 +134,12 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
     builder.append("\"").append(GatorGradleConfig.get().getAssignmentName());
     builder.append("\"").append(",");
 
+    // reflection
+    builder.append("\"reflection\":");
+    builder.append("\"").append(GatorGradleConfig.get().getReflection());
```

Comment:
This is where you would want to read the file most likely. You'll also need to escape the JSON, since if there are any JSON special characters in the reflection it'll break things.